style/styleVariables.tsx

```diff
@@ -0,0 +1,9 @@
+export const colors = {
+  first: '#59B6AE',
+  second: '#4D3771',
+  third: '#CF4160',
+  fourth: 'orange',
```

Comment:
curious -> why do you name them from 1 to 4? 😄 It's okay, I'd just like to understand e.g. when you decide to use `third`.

(also no need to keep commented code below -> let's just remove it)

Reply:
to be honest I am not sure how to name them so it's not confusing. I just found the color pallette that I liked.  Should I name them 'main', 'details' etc or 'firstpage','second page' etc?

Reply:
If `first`...`fourth` is easy for you to use in the codebase then no need for that. Usually u use the same naming as designers, e.g. `primary`/`secondary`/'extra'/`error`/`warning` color, but whatever works best for you 👍